sys/include/sched_round_robin.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2020 TUBA Freiberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sche_round_robin Round Robin Scheduler
+ * @ingroup     sys
+ * @brief       This module module provides round robin scheduling within
+ *              for all runnable threads within each not masked priority.
```

Comment:
```suggestion
 *              for all runnable threads within each unmasked priority.
```

Reply:
somehow i think "not mask" and "unmask" is not the same and further down there is a define that that implements the default value (prio 0) for the don't schedule this mask